src/res/translation/translation_zh_CN.ts

```diff
@@ -3275,62 +3275,62 @@ We muted your channel and activated &apos;Mute Myself&apos;. Please solve the fe
     <message>
         <location filename="../../util.cpp" line="1893"/>
         <source>Released under the GNU General Public License version 2 or later (GPLv2)</source>
-        <translation type="unfinished"></translation>
+        <translation>使用 GNU 通用许可证第2版或更高版本发布 (GPLv2)</translation>
     </message>
     <message>
         <location filename="../../util.cpp" line="1902"/>
         <source>This program is free software; you can redistribute it and/or modify it under</source>
-        <translation type="unfinished"></translation>
+        <translation>此程序是自由软件；您可以修改或重分发此程序，只要您遵守</translation>
     </message>
     <message>
         <location filename="../../util.cpp" line="1904"/>
         <source>the terms of the GNU General Public License as published by the Free Software</source>
-        <translation type="unfinished"></translation>
+        <translation>由自由软件基金会发布的 GNU 通用许可证的条款</translation>
     </message>
     <message>
         <location filename="../../util.cpp" line="1906"/>
         <source>Foundation; either version 2 of the License, or (at your option) any later version.</source>
-        <translation type="unfinished"></translation>
+        <translation>无论您依据的是本授权的第 2 版，或（您可选的）任一日后发行的版本。</translation>
     </message>
     <message>
         <location filename="../../util.cpp" line="1908"/>
         <source>There is NO WARRANTY, to the extent permitted by law.</source>
-        <translation type="unfinished"></translation>
```

Comment:
We once again separated once single paragraph into multiple `tr()`s which make translation become harder. Please avoid splitting one sentence into two `tr()`s :/

Reply:
Thanks for the feedback and sorry for the extra efforts.
Is it possible for you to work around this for 3.8.2?
The reason for splitting is formatting. One possible workaround for you would be to take all parts of this block, merge them (on paper), translate them and split them again in a way which makes sense for Chinese that still attempts to stay within column width limits (~70 chars). I understand that this is non-optimal form a translator point of view.

Looks like this was introduced in @8fb8ce768.
@henkdegroot Any ideas if and how we could work around this in future releases? Can you open a seperate issue for this?

cc @softins

Reply:
This update relates to displaying proper license details in the console window when using the --version.
I think this cannot be avoid, as otherwise the print in the console will not be nice.
I don't think this will be updated very often, so hope it is just a "pain" now and not anymore in the next release.

If there is a better option, please let me know and I am happy to implement.

I would advise to use Qt Linguist for the translation update, so you can see the lines which belong together.

Reply:
> Is it possible for you to work around this for 3.8.2?

Already in this PR, but no fixed line wrap alignment through.

> The reason for splitting is formatting.

This is a paragraph of legal text referring to the GPL, but not actual GPL license content, so IMHO the ~70 chars line wrap rule doesn't need to be applied here. If it is the legal text of GPL (which isn't), it shouldn't be translatable according to [here](https://www.gnu.org/licenses/translations.html).

> I think this cannot be avoid, as otherwise the print in the console will not be nice.

By the way even in a single `tr()` you can still use regular line breaks (`\n`) to make it a multi-line translatable string and also achieve the ~70 chars line wrap goal, and use the 2nd argument of `QCoreApplication::tr()`  to tell the translators to apply the ~70 chars width line wrap rule.

Reply:
@BLumia I've opened #2319 to track this. Thanks for your input!